tests/integration/testimony.test.ts

```diff
@@ -67,9 +71,11 @@ const publishTestimony = httpsCallable<
 
 let uid: string
 let user: User
+let fullName: string
 beforeEach(async () => {
   user = await signInUser1()
   uid = user.uid
+  fullName = (await getProfile(user))?.fullName || "Anonymous"
```

Comment:
Might be more appropriate to use [`??` operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing)